### DISCUSSION TITLE
[HELP] Not successful.

### DISCUSSION THREAD

**Discussion by yayalsy** [getlantern/lantern-forum-cn#904] [Q&A]
Hello developer, I have paid the relevant fees for upgrading to the professional version, but the automatic upgrade was not successful and I did not receive the activation code in my email. My account email is [yaya_lanshiya@163.com] (yaya_lanshiya@163.com) Could you please handle it? Thank you.
@Mike777Khoo

**Comment by yayalsy** [comment]
![微信图片_20240927155840](https://github.com/user-attachments/assets/b0e24104-f633-4ab4-973e-31a0856e9442)
This is the proof of my successful payment

**Comment by MaxMKMax** [comment]
@yayalsy Okay sure, i already help you submit the issues to the payment team to solve your problem. Sorry for the inconvenience caused, we will update you soon. You may also check it after a few days, if next tuesday nothing being updated, please reach me again and i will help you to re-follow up.

Thank you for supporting Lantern.
